tests/integration/teal/roundtrip/app_roundtrip_(byte[4],(bool,bool),uint64,address)[]_7_v6.teal

```diff
@@ -25,7 +25,7 @@ int 5
 extract_uint64
 store 2
 load 19
-extract 13 0
+extract 13 32
```

Comment:
So was the integration test giving a false negative?

Reply:
I don't think integration test (in general) is false negative, but the testcase is really a corner case we didn't account for.

Reply:
I think this is extracting the acddress which is at the end of the encoding, end therefore `extract 13 0` $\++$ `extract 13 32`. Does that make sense given your changes?

Reply:
Okay, this two cases are equal only under a precondition, the last element in tuple (address in this case) has no leading dynamic type(s).

If we both agree on this precondition, then yes, it makes sense given my change.